formats/install-iso.nix

```diff
@@ -5,7 +5,7 @@
   ];
 
   # for installer
-  isoImage.isoName = "nixos.iso";
```

Comment:
we could just use `lib.mkDefault` here and have people define `isoImage.isoName` in their configuration.nix if they want a different value?